client/src/App.js

```diff
@@ -1,5 +1,26 @@
+import NavFooter from './components/NavFooter';
+import { Switch, Route } from 'react-router-dom';
+import Home from './pages/Home';
+import MyShelves from './pages/MyShelves';
+import MyBooks from './pages/MyBooks';
+
 function App() {
-  return <div></div>;
+  return (
+    <div>
+      <Switch>
+        <Route path="/home">
+          <Home />
+        </Route>
```

Comment:
As you don't have _any_ props for your Route components, you could even write it like this:
```suggestion
        <Route path="/home" component={Home} />
```